create-installer.iss

```diff
@@ -0,0 +1,99 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "Odin Compiler"
+#define MyAppVersion "0.7.1"
+#define MyAppPublisher "Odin-lang Org."
+#define MyAppURL "http://www.odin-lang.com/"
+#define MyAppExeName "odin.exe"
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application.
+; Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{914BC433-99BC-495D-9C55-C8CD17DBC97C}
+AppName={#MyAppName}
+AppVersion={#MyAppVersion}
+;AppVerName={#MyAppName} {#MyAppVersion}
+AppPublisher={#MyAppPublisher}
+AppPublisherURL={#MyAppURL}
+AppSupportURL={#MyAppURL}
+AppUpdatesURL={#MyAppURL}
+DefaultDirName={pf}\Odin
+DisableProgramGroupPage=yes
+LicenseFile=E:\Odin\LICENSE
```

Comment:
Hardcoded path, seems bad?

Reply:
Ahh that was a typo from the auto-setup I used to get started, mean to be .\LICENSE